MBBSEmu.Tests/ExportedModules/Galgsbl/btuimp_Tests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Galgsbl
+{
+    public class btuimp_Tests : ExportedModuleTestBase
+    {
+        private const int BTUIMP_ORDINAL = 22;
+
+        [Fact]
+        public void btuimp_Test()
+        {
+            //Reset State
+            Reset();
```

Comment:
Reset() is only necessary for [InlineData] parameterized tests no?

Reply:
So without the reset the test fails. I think something else is happening in the `Reset()` functions on the CPU, Memory , etc. but I'm not sure what it is exactly. Doesn't hurt having it here atm.